lookout/style/typos/analyzer.py

```diff
@@ -13,16 +16,17 @@
 import pandas
 from sourced.ml.algorithms import TokenParser, uast2sequence
 
-from lookout.style.common import merge_dicts
+from lookout.style.common import load_jinja2_template, merge_dicts
 from lookout.style.format.utils import generate_comment
 from lookout.style.typos.corrector_manager import TyposCorrectorManager
-from lookout.style.typos.utils import Candidate, Columns, flatten_df_by_column
+from lookout.style.typos.utils import Candidate, Columns, flatten_df_by_column, TEMPLATE_DIR
 
 TypoFix = NamedTuple("LineFix", (
     ("head_file", File),  # file from head revision
     ("line_number", int),  # line number for the comment
-    ("token", str),  # token where typo is found
-    ("candidates", List[Candidate]),  # Candidates for token fix
+    ("identifier", str),  # identifier where typo is found
+    ("candidates", Mapping[str, Iterable[Candidate]]),  # Candidates for token fix
```

Comment:
let's make it `Sequence` because if you want to iterate more then 1 time it is better.

Also not sure about our typing agreement
Should we just specify exact types like `Dict[str, List[Candidate]]`?

@vmarkovtsev WDYT?

Reply:
@zurk, you mixing definitions of `Iterable` and `Generator` - [lmgtfy](https://nvie.com/posts/iterators-vs-generators/) in case of 1 time iterations. 
If you can find access by index or checking the length of `Iterable` - it can be narrowed down to `Sequence`. If no - no. And the same answer for similar suggestions.

Reply:
Yes, thanks for the pic. 
I still disagree because it is a List by fact because we set it to the list below, but ok, whatever.